scripts/osx-build-fresh-daedalus.sh

```diff
@@ -52,7 +52,5 @@ git clone ${URL}
 pushd daedalus
```

Comment:
I think these `*-build-fresh-daedalus.*` scripts can be dropped, really, as they're too specialized.

Reply:
OK, fixed.